src/xAccount/XAccountFactory.sol

```diff
@@ -84,31 +84,34 @@ contract XAccountFactory is Ownable2Step, Application, PortMetadata {
     }
 
     function isRegistred(address port) public view returns (bool) {
-        return REGISTRY.get(LOCAL_CHAINID(), port) != bytes4(0);
+        return bytes(REGISTRY.get(LOCAL_CHAINID(), port)).length > 0;
```

Comment:
What's the difference between `bytes(REGISTRY.get(LOCAL_CHAINID(), port)).length > 0` and `REGISTRY.get(LOCAL_CHAINID(), port).length > 0`?

Reply:
`string` has not `.length` attribution.